src/test/java/com/google/sps/MatchRankingTest.java

```diff
@@ -83,8 +83,8 @@ public void setup() {
   @Test
   public void basicRanking() {
     User currentUser = new User("");
-    currentUser.addNewBioToMapCountForTesting(user_d.getBio());
-    currentUser.addNewBioToMapCountForTesting(user_e.getBio());
+    testOnlyAddNewBio(user_d.getBio(), currentUser);
```

Comment:
observation: testOnly is usually used as a prefix for methods that might otherwise be misconstrued as part of a public interface; In this case it's a local method inside the test, so it's not particularly necessary, but lt's also not a problem if you prefer that name.

Reply:
good point, thanks!